lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "useNx": true,
-  "version": "independent"
+  "version": "1.0.0"
```

Comment:
Ah is this related to the Andrew comment/suggestion? What does this do for us?

Reply:
this change didn't seem to do anything for us, that I could tell.